### ISSUE TITLE
Message filter - alternative return code

### ISSUE THREAD

**Issue by pcause** [martinrotter/rssguard#259]
I did a small amiunt of looking at the message filters.  Nice addition.  I have a case for tt-rss where I'd like to mark a message read but tell rssguard not to keep in the database.  It looks like I can't do that.  Is this possible and I assume you could add a return code for this meaning.

I'd like the same for important as well.  I use this as a "read it later" for tt-rss as important is the starred feed and I can go there and see the read but starred items.

**Comment by martinrotter** [comment]
So basically, you want to ignore incoming message by filter (which you already can with result code `2`). But you also want that changed read/important state is propagated back to server even if the message is ignored by the filter.

I will add this soon. Also note, that this was actually already requested (but for messages which are accepted) - https://github.com/martinrotter/rssguard/issues/258.

So basically after this gets fixed, then the changed state will be synced back to server for all messages; accepted or not.

**Comment by pcause** [comment]
great and thanks

**Comment by martinrotter** [comment]
Just commited, pls wait for it to compile and then test on dev. build - then let me know. Thank you.

**Comment by pcause** [comment]
Just tested with a very simple filter 

function filterMessage() { msg.isRead = true; return 2; }

worked.  Category all marked as read.  Didn't try important yet but will in a bit.  Nice and thanks.  

-----Original Message-----
